### PR TITLE
Spotify Documentation

### PR DESCRIPTION
**Pull request by erin2722**
🎵 Here is the Spotify Documentation for the new node!

### CODE REVIEW

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - Get Related Artists
+    - Get Top Tracks
+- Player
+    - Add Song to Queue
+    - Currently Playing
+    - Next Song
+    - Pause
+    - Previous Song
+    - Recently Played
+   - Start Music
```
Comment (Shraddha2104) [inline docs/nodes/nodes-library/nodes/Spotify/README.md]: indentation a bit different

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- Playlist
+    - Add an Item
+    - Get
+    - Get Track
+   - Get User's Playlist
```
Comment (Shraddha2104) [inline docs/nodes/nodes-library/nodes/Spotify/README.md]: indentation a bit different

File: docs/nodes/credentials/Spotify/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Spotify
+You can find information about the operations supported by Spotify node on the [integrations](https://n8n.io/integrations/n8n-nodes-base.spotify) page. You can also browse the source code of the node on [Github](https://github.com/n8n-io/n8n/blob/master/packages/nodes-base/nodes/Spotify).
+
+# Authentication
+
+## Pre-requisites
+
+Create [Spotify Developer](https://developer.spotify.com/dashboard/login) account.
```
Comment (ivov) [inline docs/nodes/credentials/Spotify/README.md]: `Create a`

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    - Add Song to Queue
+    - Currently Playing
+    - Next Song
+    - Pause
+    - Previous Song
+    - Recently Played
+   - Start Music
+- Playlist
+    - Add an Item
+    - Get
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Spotify/README.md]: I would expand: `Get a Playlist`

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    - Get Track
+   - Get User's Playlist
+    - Remove an Item
+- Track
+    - Get
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Spotify/README.md]: I would expand: `Get a Track`

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    - Get Audio Features
+
+
+## Example Usage
+
+This workflow allows you to add a song to your queue in Spotify. You can also find the [workflow](https://n8n.io/workflows/440) on the website. This example usage workflow would use the following two nodes.
+- [Start](../../core-nodes/Start/README.md)
+- Spotify
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Spotify/README.md]: I think the latest directive is that this has to be an empty link: `[Spotify]()`

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 7]
```diff
@@ -11,37 +11,37 @@ You can find authentication information for this node [here](../../../credential
 ## Basic Operations
 
 - Album
-    - Get
-    - Get Tracks
+    - Get an Album
+    - Get an Album's Tracks
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Spotify/README.md]: Most docs under `Basic Operations` seem to use uppercase only for the first letter. This is unlike resource operation names in the codebase, where the casing is exactly as it is here. Please double check.

**Review by ivov — CHANGES_REQUESTED** [review]

**Review by ivov — APPROVED** [review]

**Review by ivov — APPROVED** [review]

**Comment by ivov** [comment]
Submitted and merged: https://github.com/n8n-io/n8n-docs/pull/15

**Pull request closed without merging**